src/pages/add/select-currency/select-currency.ts

```diff
@@ -191,4 +211,62 @@ export class SelectCurrencyPage {
       this.events.publish('Local/WalletListChange');
     });
   }
+
```

Comment:
Should we create a separate `select-token` component and move token/wallet specific logic to that? Then in the `select-currency.html` replace the token html section to `<SelectToken wallets=wallets></SelectToken>` component?